lib/experimental/new_embed.dart

```diff
@@ -109,9 +118,10 @@ class NewEmbed {
 
   NewEmbed(this.options) {
     _initHostListener();
-    tabController = NewEmbedTabController();
+    tabController =
+        NewEmbedTabController(MDCTabBar(querySelector('.mdc-tab-bar')));
 
-    var tabNames = ['editor', 'test', 'solution'];
+    var tabNames = ['editor', 'solution', 'test'];
```

Comment:
🤔

Reply:
the order needs to match the order they appear in the HTML, otherwise the wrong tab is selected when using the mdc_web tab bar UI

Reply:
Ah, got it. Makes more sense than "the order in which the tabs were added to the codebase. 😄